cmake-modules/gepetto-viewer-corba-plugin.cmake

```diff
@@ -78,3 +78,17 @@ MACRO(GEPETTO_GUI_PLUGIN PLUGIN_NAME)
   # Uncomment to install
   INSTALL(TARGETS ${PLUGIN_NAME} DESTINATION lib/gepetto-gui-plugins)
 ENDMACRO()
+
+# Declare a Python plugin
+MACRO(GEPETTO_GUI_DECLARE_PYPLUGIN file)
+  STRING(MD5 file_md5 "${PYTHON_SITELIB}/${file}")
```

Comment:
I don't get why you are using the hash of the file as a name

Reply:
That was the easiest way I found to get a *unique* name from an absolute filename. I could have tried to replace all `/` by `_` but that is neither safe nor convenient (although more convenient that hashes). I don't think it's the best option.

The requirement here is simple: don't override an existing file except if the existing file was generated by the same call. I am open to other methods.